save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DemoManagerController.kt

```diff
@@ -100,16 +101,18 @@ class DemoManagerController(
                 }
                 .toBodilessEntity()
         }
-        .flatMap {
+        .map {
```

Comment:
also suggestion to collect all required info to a single request and send it to demo (in `doOnSuccess` section) and here return a single `accepted`

Reply:
the same as demo can fail when file started to upload.
but got you point and actually, Demo can return 202 on its side.
As I can see -- backend is just a proxy here (to check authorization and authentication).
So common suggestion to reduce logic on backend and move it to demo

Reply:
ok, may be done in next PR